apps/kampus/package.json

```diff
@@ -15,6 +15,7 @@
     "relay-compiler": "15.0.0"
   },
   "dependencies": {
+    "@kampus/email": "0.0.0",
```

Comment:
Official semver docs does recommend starting at 0.1.0

https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

Reply:
Actually if it's internal (monorepo) package it should be "@kampus/email: *"

Reply:
Interesting. I always try to version my packages, no matter if they're internal or not. Why do you suggest using `*` as a version number?

Reply:
[Check here](https://turbo.build/repo/docs/handbook/sharing-code/internal-packages#what-makes-a-package-internal) , internal packages are not versioned or published. If you change internal package to version "0.2.0" but one of your app uses "0.1.0" which is not something published or can be accessed.

Reply:
Their suggestion to use "*" as well https://turbo.build/repo/docs/handbook/sharing-code/internal-packages#3-import-the-package

Reply:
I didn't know that. Thank you 🙏

Reply:
> Actually if it's internal (monorepo) package it should be "@kampus/email: *"

wait, it is `"@kampus/email": "*" right?

Reply:
BTW, ty so much for the explanation @Ketcap i will open a PR to fix this in each package.